packages/wheelhouse/src/helmActions.js

```diff
@@ -0,0 +1,37 @@
+import { safeLoad as parseYaml } from "js-yaml";
+import { resolve } from "path";
+import fs from "fs-extra";
+import debug from "debug";
+import { HELM_LOADED, PACKAGES_CHECK_SYNC } from "wheelhouse-core";
+
+const log = debug("wheelhouse:packagesActions");
+
+export const helmLoad = pkgPath => async (dispatch, getState) => {
+  log(`Loading ${pkgPath}`);
+  const { rootDir } = getState().config;
+  const pkgHelmPath = resolve(rootDir, pkgPath, "Chart.yaml");
+  try {
+    await fs.stat(pkgHelmPath);
+  } catch (err) {
+    if (err.code === "ENOENT") {
+      // This package doesn't have a Chart.yaml and that's just fine.
+      return;
+    }
+    log(`Unexpected error attempting to stat ${pkgHelmPath}`, err);
+    throw err;
+  }
+  const yamlFile = await fs.readFile(pkgHelmPath, "utf8");
+  const pkg = parseYaml(yamlFile);
+  await dispatch(
+    helmLoaded({
+      chartYaml: pkg
+    })
+  );
+  await dispatch({
```

Comment:
does it need to be dispatched from the reducer? i moved it to developmentLint, seems to be more appropriate there, but I can move it to the reducer if you think that's better.